src/tests/ftest/osa/osa_online_reintegration.py

```diff
@@ -55,7 +53,8 @@ def daos_racer_thread(self):
         self.daos_racer.run()
 
     def run_online_reintegration_test(self, num_pool, racer=False,
-                                      server_boot=False):
+                                      server_boot=False,
+                                      oclass=None):
```

Comment:
You might want to update the argument descriptions.

Reply:
Done.